ajax/libs/qartjs/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "qartjs",
+  "version": "1.0.2",
+  "description": "qart.js merge picture and QR code.",
+  "keywords": [
+    "qart",
+    "qr",
+    "qrcode"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kciter/qart.js.git"
+  },
+  "author": "Lee Sun-Hyoup <kciter@naver.com>",
+  "license": "GPL-3.0",
+  "npmName": "qartjs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "qart.min.js",
+  "autoupdate": {
```

Comment:
@maruilian11 Sorry about that - I think rebased incorrectly. Will update in a few, thanks.